tests/test_fractional_octaves.py

```diff
@@ -0,0 +1,146 @@
+from unittest import mock
+
+import numpy as np
+import pytest
+from numpy import testing as npt
+import scipy.signal as spsignal
+
+from pyfar.dsp import filter
+from pyfar.dsp.classes import FilterSOS
+from pyfar.signal import Signal
+
```

Comment:
General Question: Do we want to add a test for the summed frequency response? This is what DIN 61260 demands:

4.9 Summierung von Ausgangssignalen
Für ein sinusförmiges Eingangssignal beliebiger Frequenz zwischen den Bandmittenfrequenzen zweier
benachbarter Oktaven oder Bruchteilen von Oktaven darf die Differenz zwischen
a) dem um die Referenzdämpfung verringerten Pegel des Eingangssignals und
b) dem Pegel der Summe der zeitlich quadratisch gemittelten Ausgangssignale verschiedener Filter mit
angegebener Filterbandbreite für Geräte der Klassen 0, 1 oder 2 den Bereich von ± 1,0 dB; + 1,0 dB,
-2,0 dB beziehungsweise + 2,0 dB, -4 dB nicht überschreiten.